go.sum

```diff
@@ -1,8 +1,18 @@
 github.com/OneOfOne/xxhash v1.2.2/go.mod h1:HSdplMjZKSmBqAxg5vPj2TmRDmfkzw+cTzAElWljhcU=
 github.com/OneOfOne/xxhash v1.2.7 h1:fzrmmkskv067ZQbd9wERNGuxckWw67dyzoMG62p7LMo=
 github.com/OneOfOne/xxhash v1.2.7/go.mod h1:eZbhyaAYD41SGSSsnmcpxVoRiQ/MPUTjUdIIOT9Um7Q=
+github.com/alecthomas/template v0.0.0-20160405071501-a0175ee3bccc/go.mod h1:LOuyumcjzFXgccqObfd/Ljyb9UuFJ6TxHnclSeseNhc=
```

Comment:
WOW!  This brings in a lot of dependencies!

Reply:
NIT: Also, consider running `go mod tidy` to clean up some of these.  Ex, `go-spew` is still listed in this file.

Reply:
this actually is the output of `go mod tidy`. not sure where `go-spew` is coming from.

Reply:
yikes!  Something else may be using it.  it is pulling in `go-kit` and `grpc`.  eithe one of those could have that as a dependency.  `go mod graph | grep go-spew` would tell you what's using it.

Reply:
something seems to be using testify and logrus which have go-spew as dependencies